internal/karma/karma.conf.js

```diff
@@ -1,5 +1,59 @@
 // Karma configuration
 // GENERATED BY Bazel
+const path = require('path');
+const fs = require('fs');
+const tmp = require('tmp');
+
+process.env.CHROME_BIN = require('puppeteer').executablePath();
```

Comment:
in theory the `ts_web_test` should allow the user to select one or more browsers,
https://github.com/bazelbuild/rules_typescript/blob/master/internal/karma/ts_web_test.bzl#L22

just to keep in mind, that anything hard-coded will need to be revisited

Reply:
I'll add a comment to that line explaining it a bit. Karma will just look for CHROME_BIN in the environment if it is configured to use Chrome. If another browser is selected then that value is not used.